### PR TITLE
Extract BMP magic into separate define

### PR DESCRIPTION
**Pull request by DJm00n**
Minor cleanup change.

### CODE REVIEW

File: src/video/windows/SDL_windowsclipboard.c [hunk 1]
```diff
@@ -35,12 +35,11 @@
 
 #define IMAGE_FORMAT CF_DIB
 #define IMAGE_MIME_TYPE "image/bmp"
+#define BFT_BITMAP 0x4d42 /* 'BM' */
```
Comment (DJm00n) [inline src/video/windows/SDL_windowsclipboard.c:38]: Well-known define:
https://github.com/search?q=repo%3Amicrosoft%2FWindows-classic-samples%20BFT_BITMAP&type=code

File: src/video/windows/SDL_windowsclipboard.c [hunk 2]
```diff
@@ -69,7 +68,7 @@ static HANDLE WIN_ConvertBMPtoDIB(const void *bmp, size_t bmp_size)
 {
     HANDLE hMem = NULL;
 
-    if (bmp && bmp_size > sizeof(BITMAPFILEHEADER) && SDL_memcmp(bmp, bmp_magic, sizeof(bmp_magic)) == 0) {
+    if (bmp && bmp_size > sizeof(BITMAPFILEHEADER) && ((BITMAPFILEHEADER *)bmp)->bfType == BFT_BITMAP) {
```
Comment (slouken) [inline src/video/windows/SDL_windowsclipboard.c:71]: This assumes a little-endian machine, which I think is fine, but maybe better not to leave that footgun here? If you really want to do this, you should probably use the SDL endian swap functions to guarantee the value.

Comment (sezero) [reply src/video/windows/SDL_windowsclipboard.c:71]: Windows has always been little endian, even when they had played with ppc back in the day, so it should be OK?

Comment (slouken) [reply src/video/windows/SDL_windowsclipboard.c:71]: Good point. Okay, I'll merge as-is.

**Pull request merged**